guides/common/modules/con_upgrading-smartproxyservers-in-a-load-balancing-setup.adoc

```diff
@@ -0,0 +1,7 @@
+[id="upgrading-smartproxyservers-in-a-load-balancing-setup_{context}"]
+= Upgrading {SmartProxyServers} in a load balancing setup
```

Comment:
```suggestion
= Upgrading {SmartProxyServers} in a load-balancing setup
```

Unless we have a style guideline not to hyphenate "load-balancing" when used as a compound adjective, this should probably be hyphenated.